### ISSUE TITLE
I don't have adapater_config.json and adapter_model.bin. after finishing training

### ISSUE THREAD

**Issue by hotco87** [huggingface/peft#345]
I run this finetune.py from  "https://github.com/tloen/alpaca-lora"

After training, I got 
**optimizer.pt pytorch_model.bin rng_state.pth scaler.pt scheduler.pt trainer_state.json training_args.bin**

I don't have **both adapater_config.json and adapter_model.bin.**

**Comment by hotco87** [comment]
After all training is finished, adapter_config.json and adapter_model.bin are created.

**Comment by tiantao911** [comment]
After all training is finished, I don't have both adapater_config.json and adapter_model.bin.
